lib/krikri/provider_list.rb

```diff
@@ -0,0 +1,19 @@
+module Krikri
+  # Contrtructs a list of providers
+  class ProviderList
+
+    def initialize
+      @index_querier = Krikri::IndexQuerier.new
+      @default_params = { :rows => 10,
```

Comment:
Sure - do you think I should also change `@default_params` to a class-level constant in `RandomRecordGenerator` and `ValidationReportList`?